status-im/status-react
# chore: remove old wallet code, signing code and keycard code

fixes: https://github.com/status-im/status-mobile/issues/18559

This pr deletes numerous other parts of the legacy code. I could not find any of these screens as a result of the legacy wallet already being deleted.

This also removes the keycard code as we are currently not supporting the legacy implementation anyway and the new implementation will have to be revisited.

It removes all signing and wallet connect pages for the same reason as keycard.

To test:
A quick happy path pass through the app and check some of the legacy settings particularly related to wallet/ network and keycard signing etc. 
It is also good if a quick check of the browser can be done.
